compiler/circle-quantizer/src/CircleQuantizer.cpp

```diff
@@ -38,8 +38,40 @@ using LayerParams = luci::CircleQuantizer::Options::LayerParams;
 using Algorithms = luci::CircleQuantizer::Options::Algorithm;
 using AlgorithmParameters = luci::CircleQuantizer::Options::AlgorithmParameters;
 
+struct QConfReader
+{
+  void init(std::string &filename)
+  {
+    _ifs.open(filename);
+
+    // Failed to open cfg file
+    if (not _ifs.is_open())
+      throw std::runtime_error("Cannot open config file. " + filename);
+
+    JSONCPP_STRING errs;
+    Json::CharReaderBuilder builder;
+
+    // Failed to parse
+    if (not parseFromStream(builder, _ifs, &_root, &errs))
+      throw std::runtime_error("Cannot parse config file (json format). " + errs);
```

Comment:
It would be better to close `_ifs` right after parsing, rather than in destructor.